dev/django-public.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8
```

Comment:
Would `-slim` (165M) or `-alpine` (43M) be enough?  python:3.8 is quite heavier (800M?)

Reply:
`-slim` works.